linux/LinuxProcessList.c

```diff
@@ -626,6 +626,30 @@ static void LinuxProcessList_readCtxtData(LinuxProcess* process, const char* dir
    process->ctxt_total = ctxt;
 }
 
+static void LinuxProcessList_readSecattrData(LinuxProcess* process, const char* dirname, const char* name) {
+   char filename[MAX_NAME+1];
+   xSnprintf(filename, MAX_NAME, "%s/%s/attr/current", dirname, name);
```

Comment:
Use `sizeof(filename)`, no need to account for that final `\0`, already done by `xSnprintf`.

Reply:
yep, fixed